contracts/TokenFactory/TokenFactoryFacet.sol

```diff
@@ -23,7 +41,7 @@ contract TokenFactoryFacet is ITokenFactory {
     ) external override {
         LibDiamond.enforceIsContractOwner();
 
-        TokenLimitedSupply t = new TokenLimitedSupply(_name, _symbol, to, amount);
+        LimitedSupplyToken t = new LimitedSupplyToken(_name, _symbol, to, amount);
```

Comment:
I was also doubting to do this rename but be aware the name is used in quite some places, al least in deployment, write-abi script, and in the API project... So tread carefully!

Reply:
It also breaks backwards compatibility, will you release it as 3.0.0? :)